packages/shared/src/hooks/notifications/useBookmarkReminder.ts

```diff
@@ -63,6 +68,20 @@ export const useBookmarkReminder = (): UseBookmarkReminder => {
         updatePostCache(client, postId, (post) => ({
           bookmark: { ...post.bookmark, remindAt },
         }));
+
+        if (feedQueryKey) {
```

Comment:
Just something to really test thorogouly, could easily creep in issues that are hard to spot.

Reply:
We should be safe here, this was taken from the process of `useBookmarkPost` - the only difference is the object we are trying to pass for update.